pl/android_release.lang

```diff
@@ -12,7 +12,7 @@ Przeglądarka Firefox
 
 # This is the short tagline for the app on Google Play
 ;Get the customizable, private & free mobile browser that syncs across devices.
-Get the customizable, private & free mobile browser that syncs across devices.
+Pobierz konfigurowalną, szanującą prywatność, darmową przeglądarkę, którą można synchronizować ze wszystkimi urządzeniami.
```

Comment:
This string has 80 characters limit, see https://groups.google.com/forum/m/#!topic/mozilla.dev.l10n.web/ftvjEgBGQy0